cr/api.py

```diff
@@ -478,15 +478,15 @@ def api_get_task(self, task_id: int) -> dict:
 
     def api_poll_task(self, task_id: int) -> dict:
         """
-        Blocking function to poll a task every 10 seconds until it completes.
+        Blocking function to poll a task every 10 seconds until it completes or errors out.
```

Comment:
Minor nit pick, but can you wrap this to make sure it's within 80 character line length? Thought ruff might have done it but I guess not being a comment.